contracts/tribute/src/testing.rs

```diff
@@ -1179,3 +1202,115 @@ fn test_query_outstanding_tribute_claims() {
         }
     }
 }
+
```

Comment:
do we even still need this then if we have the other test case you added? do you think it adds value?

Reply:
ok, let's remove it!

Reply:
Done in https://github.com/informalsystems/hydro/pull/129/commits/9a252f41b8a55c04be3951b9ea7cb3ea1f0ca072